klippa/src/lib.rs

```diff
@@ -394,6 +449,30 @@ fn get_font_num_glyphs(font: &FontRef) -> usize {
     ret.max(maxp.num_glyphs() as usize)
 }
 
+fn remap_indices<T: Domain + std::cmp::Eq + std::hash::Hash>(
+    indices: IntSet<T>,
+) -> FnvHashMap<T, T> {
+    indices
+        .iter()
+        .enumerate()
+        .map(|x| (x.1, T::from_u32(InDomain::new(x.0 as u32))))
```

Comment:
InDomain construction is meant to be only done internally in IntSet. I think you can add the From<u32> trait to T to allow you to construct one from a u32 in this function.